main/src/ca/uwaterloo/flix/language/ast/TokenKind.scala

```diff
@@ -426,6 +426,17 @@ sealed trait TokenKind {
     case _ => false
   })
 
+  /**
+    * Checks if this token is one of the [[TokenKind]]s that can validly appear as the first token in a declaration within an effect.
+    * Note that a CommentDoc, a Modifier and/or an annotation may lead such a declaration.
```

Comment:
I copy-pasted this.

The comment about modifier is wrong. But now its consistently wrong with all the `isFoo`.